pallets/connectors/src/message.rs

```diff
@@ -56,13 +65,72 @@ where
 		address: Address,
 		valid_until: Moment,
 	},
+	// Bidirectional: Domain must not accept every incoming token.
+	// Sender must ensure beforehand that the receiver will not reject
```

Comment:
> // Sender must ensure beforehand that the receiver will not reject

This is not technically possible; As in, `sender` can't check with the receiver that the receiver will not reject handling a given token. I think the point of that sentence is to say that Sender can't ensure that receiver will handle it, so sender should make sure to only send tokens it knows that the receiver can handle. Since transfers are only to happen between Centrifuge and an EVM domain for now, this means that Centrifuge should make sure to have called `AddCurrency(C, _)` before `Transfer(C, _)`; The Transfer from a Domain down to Centrifuge should always work because if Domain as able to send `C`, it was by definition added already on that domain.

Reply:
Thanks for the explanation, makes absolute sense! This requirement was copy pasted [from the spec](https://centrifuge.hackmd.io/Soy4JiFbRSyovj53KW-a4A?view#Transfer) btw.

Reply:
Yeah I noticed the source of that line :) my pleasure!